UIKitDZ/UIKitDZ/StopWatchModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  StopWatch.swift
+//  UIKitDZ
+//
+//  Created by Max Inedom on 7/9/21.
+//
+
+import Foundation
+
+final class StopWatchModel {
+    
+    //MARK: - Private Properties
+    private var startTime: NSDate?
+    private var elapsedTime: TimeInterval {
+        if let startTime = self.startTime {
+            return -startTime.timeIntervalSinceNow
+        } else {
+            return 0
+        }
+    }
+    
+    //MARK: - Public Properties
+    public var isRunning: Bool {
```

Comment:
```suggestion
    var isRunning: Bool {
```
public не пишем никогда

Reply:
изменил